compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -9933,4 +9945,10 @@ public static class AnalyzerData {
         QueryTypeChecker.AnalyzerData queryData = new QueryTypeChecker.AnalyzerData();
         Set<String> queryVariables;
     }
+
+    /**
```

Comment:
Add documentation (if we continue to keep this).

Reply:
Added